alfasim_sdk/hook_specs.py

```diff
@@ -146,6 +146,20 @@ def update_plugins_secondary_variables(ctx: "void*") -> "int":
     """
 
 
+def update_plugins_secondary_variables_on_first_timestep(ctx: "void*") -> "int":
```

Comment:
@williamjamir asked to review all hook names. This will be done on a future PR, when we have more information about what the plugins actually need.